1_fetch/src/get_nlcd_LC.R

```diff
@@ -1,7 +1,8 @@
 ###-----------------------------------------------------------------------------
-download_NHD_NLCD_data <- function(sb_id,
+download_NHD_data <- function(sb_id,
                                  out_path = '1_fetch/out',
                                  downloaded_data_folder_name = NA,
+                                 output_data_parent_folder = NA,
```

Comment:
It's generally better to use NULL here for empty objects. But I'm not sure how you're using NA in the following code. You can use `is.null` just like `is.na`